src/main/java/frc/robot/commands/AimBot.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Robot;
+
+public class AimBot extends CommandBase
+{
+    @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+    private Robot robot;
+
+    public AimBot(Robot robot)
+    {
+        this.robot = robot;
+        addRequirements(robot.drivetrain, robot.vision, robot.shooter);
```

Comment:
I still think it would be clearer if you had multiple `addRequirements`, one for each subsystem, but up to you.